Source/Charts/Data/Interfaces/IChartDataSet.swift

```diff
@@ -27,73 +27,77 @@ public protocol IChartDataSet
     /// This is only needed for the autoScaleMinMax feature.
     func calcMinMaxY(fromX: Double, toX: Double)
     
-    /// - returns: The minimum y-value this DataSet holds
+    /// The minimum y-value this DataSet holds
     var yMin: Double { get }
     
-    /// - returns: The maximum y-value this DataSet holds
+    /// The maximum y-value this DataSet holds
     var yMax: Double { get }
     
-    /// - returns: The minimum x-value this DataSet holds
+    /// The minimum x-value this DataSet holds
     var xMin: Double { get }
     
-    /// - returns: The maximum x-value this DataSet holds
+    /// The maximum x-value this DataSet holds
     var xMax: Double { get }
     
-    /// - returns: The number of y-values this DataSet represents
+    /// The number of y-values this DataSet represents
     var entryCount: Int { get }
     
-    /// - returns: The entry object found at the given index (not x-value!)
-    /// - throws: out of bounds
+    /// - Throws: out of bounds
     /// if `i` is out of bounds, it may throw an out-of-bounds exception
+    /// - Returns: The entry object found at the given index (not x-value!)
```

Comment:
do we need to add `///` here?

Reply:
This is the complete code:

```swift
/// - Throws: out of bounds
/// if `i` is out of bounds, it may throw an out-of-bounds exception
/// - Returns: The entry object found at the given index (not x-value!)
func entryForIndex(_ i: Int) -> ChartDataEntry?
```

I regard line 46 as part of "Throws" section, so I did not add and separator line here.